### DISCUSSION TITLE
Wifi Unsafe Noises

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#310] [Announcements]
### New feature(s)
* New API calls for plugin developers:
  * is_file_loaded
  * is_mod_enabled
  * show_config_editor
  * show_main_menu

Relevant changes have been performed on [arcropolis-api](https://github.com/Raytwo/arcropolis_api) to account for these.

### Change(s)
* Reword the popup prompt when generating a new configuration on first boot.

### Special thanks
* itsmeft, for ``arcrop_is_file_loaded``.
* Kalomaze, for rewording the popup prompt.

# Read this
Skyline is not be shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.3.0'>Wifi Safe Noises</a>.</em>

**Comment by mRama3415** [comment]
Ever since i updated to 3.3.0, my ps2 stage will drop frames randomly even without any mods on the stage. Is there a fix to this?

**Reply by Raytwo** [reply]
The changes from this version have no impact on anything loading related, so if such an issue exists, it’d have been in previous versions too.
